compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -408,12 +409,17 @@ modInstanceInfoFromDetails :: ModDetails -> ModInstanceInfo
 modInstanceInfoFromDetails ModDetails{..} = MS.fromList
     [ (is_dfun, overlapMode is_flag) | ClsInst{..} <- md_insts ]
 
-convertInterfaces :: Env -> [TyThing] -> ConvertM [Definition]
-convertInterfaces env tyThings
+interfaceNames :: Env -> [TyThing] -> ConvertM (S.Set T.Text)
```

Comment:
This doesn't need to be in ConvertM, as far as I can tell.

Reply:
dropped it